example/deploy/crds/mellanox.com_macvlannetworks_crd.yaml

```diff
@@ -0,0 +1,90 @@
+# Copyright 2020 NVIDIA
```

Comment:
just a note, any changes if applied to this CRD in `deploy/` should be copied to examples for all CRDs/CRs under `example/deploy`